README.md

```diff
@@ -1,19 +1,24 @@
-# LaminR: Work with LaminDB instances in R
+# laminr: An R interface to LaminDB
 
-
-<!-- 
-DO NOT edit the README.md directly.
-&#10;Instead, edit the README.qmd file and render it using `quarto render README.qmd`. 
--->
 <!-- badges: start -->
-
 [![R-CMD-check](https://github.com/laminlabs/laminr/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/laminlabs/laminr/actions/workflows/R-CMD-check.yaml)
 <!-- badges: end -->
 
-This package allows you to query and download data from LaminDB
-instances.
+**laminr** is an R package that provides an interface to LaminDB, a powerful open-source data framework designed specifically for biological research. LaminDB enables you to manage, query, and track your data and metadata in a unified and scalable way.
```

Comment:
There is some inconsistency in how the package name is formatted in different places. I tend to use these conventions but I just made them up:

* **{package}** - An R package
* `package::function()` - A function in an R package
* ***package*** or **package** - A Python package
* `package.function()` - A function in a Python package
* **Emphasised** or _Emphasised_ - Some other important concept
* `code` - Other parts of code including objects, variables etc. This is also
  used for files or directories.